<?xml version="1.0" ?>
<launch>
  <arg name="output"        default="screen"/>

  <node pkg="drone_odometry" type="px4_tf_publisher" name="px4_tf_publisher" output="$(arg output)" >
    <!-- <remap from="/odometry/filtered" to="/camera/odom/sample"/> -->
    <!-- <remap from="/odometry/filtered" to="/mavros/odometry/out"/> --> <!-- use companoion pc ekf-->
    <remap from="/odometry/filtered" to="/mavros/odometry/in"/>  <!-- use drone ekf-->

    <param name="cov_tresh_critical" value="0.08"/>  
    <param name="cov_tresh_termination" value="0.3"/> 
    <param name="odom_timeout" value="0.1"/>   
    
    <param name="map_frame" value="map"/>  
    <param name="odom_frame" value="odom"/>  
    <param name="base_aux_frame" value="base_link_stabilized"/>  
    <param name="base_frame" value="base_link"/>  
    <param name="tf_pub_rate" value="30.0"/>   <!-- 10 -->
    <param name="publish_tf" value="true"/>  

  </node>

</launch>
